recipes/pdfcombine/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pdfcombine" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "23e3af619a4859008745ac61a6040a230023cd4e5c4c667d9b8ea248726f47ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
Using `fn` is not required anymore, so please remove that line. When not used, the tarball is saved as `v0.3.1.tar.gz` as specified with the last part of `url`.

Reply:
Thanks, did not realise